integration_tests/CMakeLists.txt

```diff
@@ -456,6 +476,7 @@ RUN(NAME bindc_05   LABELS llvm c
         EXTRAFILES bindc_05b.c)
 RUN(NAME bindc_06   LABELS llvm c
         EXTRAFILES bindc_06b.c)
+RUN(NAME bindpy_01           LABELS c ENABLE_CPYTHON EXTRAFILES bindpy_01_module.py)
```

Comment:
Does this work with `cpython`? We should try to implement it, to make sure all the syntax works.